modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/GranuleAccessProvider.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2020, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.gce.imagemosaic;
+
+import java.io.IOException;
+import javax.imageio.ImageReader;
+import javax.imageio.spi.ImageInputStreamSpi;
+import javax.imageio.spi.ImageReaderSpi;
+import javax.imageio.stream.ImageInputStream;
+import org.geotools.coverage.grid.io.AbstractGridCoverage2DReader;
+import org.geotools.coverage.grid.io.AbstractGridFormat;
+import org.geotools.coverage.grid.io.imageio.MaskOverviewProvider;
+import org.geotools.util.factory.Hints;
+
+/**
+ * An interface providing multiple types of entities involved when getting access to a Granule.
+ *
+ * <p>An ImageReaderSpi to create an ImageReader to decode that granule An ImageInputStreamSpi to
```

Comment:
I guess this was meant to be a list, use <ul><li>item</li>...</ul> to avoid the formatter from squashing it in one line.

Reply:
Yep. Not sure why it added a <p>. 
Your comments come parsed by github :) I will use a UL and LI Items :)